src/components/BreadCrumb/BreadCrumb.module.scss

```diff
@@ -0,0 +1,7 @@
+/* BreadCrumb.module.scss */
+.breadcrumb {
+  display: flex; /* Arrange items inline */
+  align-items: center; /* Vertically center items */
+  font-size: 14px;
+  color: #555; /* Default text color */
```

Comment:
the color should not be hard-coded, in-stead it would be better if it can be accessed from theme.  similar for font-size.